### PR TITLE
[CFM-929] Related Groups block on group dashboard

### PR DESCRIPTION
**Pull request by guytepper**
**JIRA:** https://issuetracker.amplexor.com/jira/browse/CFM-929

**Mobile:**
image

**Tablet:**
image

**Desktop:**
image

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/views/handlers/c4m_overview_groups_handler_members_count.inc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+/**
+ * Handler for the group's members count custom field.
+ *
+ * @ingroup views_field_handlers
+ */
+class c4m_overview_groups_handler_members_count extends views_handler_field_entity {
+  
+  function render($values) {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/views/handlers/c4m_overview_groups_handler_members_count.inc]: please add docs

File: project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/views/handlers/c4m_overview_groups_handler_members_count.inc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    $query = new EntityFieldQuery();
+    return $query
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/views/handlers/c4m_overview_groups_handler_members_count.inc]: add a line spacing above

File: project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/views/handlers/c4m_overview_groups_handler_members_count.inc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      ->entityCondition('entity_type', 'og_membership')
+      ->propertyCondition('group_type', 'node')
+      ->propertyCondition('gid', $values->nid)
+      ->propertyCondition('entity_type', 'user')
+      ->count()
+      ->execute();
+  }
+
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/views/handlers/c4m_overview_groups_handler_members_count.inc]: remove extra line

File: project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/views/handlers/c4m_overview_groups_handler_members_count.inc [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+/**
+ * Handler for the group's members count custom field.
+ *
+ * @ingroup views_field_handlers
+ */
+class c4m_overview_groups_handler_members_count extends views_handler_field_entity {
+
+  /**
+   * @param $node
+   * @return mixed|null
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/views/handlers/c4m_overview_groups_handler_members_count.inc]: @guytepper please fix the docs.

Comment (guytepper) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/views/handlers/c4m_overview_groups_handler_members_count.inc]: Is it ok now?

File: project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/views/handlers/c4m_overview_groups_handler_members_count.inc [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+class c4m_overview_groups_handler_members_count extends views_handler_field_entity {
+
+  /**
+   * @param $node
+   *  The group to check it's member's count.
+   *
+   * @return int|null
+   *  The group's members count, if the node provided is valid.
+   *
+   * Uses the current group ID to fetch & return it's member's count.
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/views/handlers/c4m_overview_groups_handler_members_count.inc]: fix the docs

Comment (guytepper) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/views/handlers/c4m_overview_groups_handler_members_count.inc]: What's wrong?

**Pull request merged**